### PR TITLE
GDB-9198 allow WB user to provide context link

### PR DESCRIPTION
**Pull request by VPetrova00**
### **What**
New setting property in Import Settings view.

### **Why**
In order to be easier for the user to add JSONLD context link through the WB.

### **How**
- Added html element for the new property.
- Changed the import template in order to include the new property `contextLink`.
- Created a cypress test in order to validate the changes.

Screenshot 2023-11-24 at 11 06 01

### CODE REVIEW

File: src/js/angular/import/templates/settingsModal.html [hunk 1]
```diff
@@ -5,264 +5,301 @@ <h4 class="modal-title">{{'import.settings' | translate}}</h4>
 <div class="modal-body">
     <form name="settingsForm" id="settingsForm" class="settings-form">
         <div class="form-horizontal">
-        <div class="form-group row">
```
Comment (svilenvelikov) [inline src/js/angular/import/templates/settingsModal.html:8]: Fix the formatting please. Check if your IDE isn't configured to format the whole file. Revert the file and reapply the changes without formatting the entire file.

**Comment by svilenvelikov** [comment]
Also, please follow the MR description rules. Check here for example https://github.com/Ontotext-AD/graphdb-workbench/pull/1144

File: src/js/angular/import/templates/settingsModal.html (hunk 1 above)
Comment (VPetrova00) [reply src/js/angular/import/templates/settingsModal.html:8]: I am not sure if I successfully reverted the file format but the IDE shows me that this is the previous format with my change.

Comment (VPetrova00) [reply src/js/angular/import/templates/settingsModal.html]: Fixed file alignment.

**Comment by svilenvelikov** [comment]
I just wonder if you discussed in which position this new context link field should be placed in the form? Because IMO it should sit somewhere below base IRI or target graph fields.

File: src/i18n/locale-en.json [hunk 2]
```diff
@@ -925,6 +926,7 @@
     "import.last.import.settings": "Last import settings",
     "import.mode.not.supported.constraint": "This mode is not supported when importing multiple items.",
     "import.enable.replace.option": "Enable this to replace the data in one or more graphs with the imported data.",
+    "import.context.link": "Context link",
```
Comment (svilenvelikov) [inline src/i18n/locale-en.json:929]: You should also update the file `test-cypress/fixtures/locale-en.json`

**Comment by svilenvelikov** [comment]
Would you also consider writing additional or updating some of the existing cypress tests to cover this change

File: src/i18n/locale-en.json (hunk 2 above)
Comment (VPetrova00) [reply src/i18n/locale-en.json:929]: Updated.

File: test-cypress/integration/import/import.server.files.spec.js [hunk 3]
```diff
@@ -42,6 +44,22 @@ describe('Import screen validation - server files', () => {
             .verifyImportStatusDetails(FILE_FOR_IMPORT, [CONTEXT, BASE_URI, '"preserveBNodeIds": true,']);
     });
 
+    // for this test it is necessary to set up a whitelist to GraphDB in this way: -Dgraphdb.jsonld.whitelist=<absolute_path_to_fixtures/graphdb-import>
+    it.only('Test import Server files successfully with JSONLD context link settings', () => {
```
Comment (svilenvelikov) [inline test-cypress/integration/import/import.server.files.spec.js:48]: Please remove the `only`

Comment (VPetrova00) [reply test-cypress/integration/import/import.server.files.spec.js]: Removed.

File: .env [hunk 4]
```diff
@@ -1 +1 @@
-GDB_VERSION=10.5.0-TR1
+GDB_VERSION=10.5.0-RC1
```
Comment (svilenvelikov) [inline .env:1]: What about updating this version? This one is not relevant anymore

**Review by svilenvelikov — APPROVED** [review]

**Comment by VPetrova00** [comment]
Jenkins retry build

**Pull request merged**